src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ProcessApiServiceImpl.scala

```diff
@@ -399,29 +399,6 @@ final case class ProcessApiServiceImpl(
     }
   }
 
-  override def updateEServiceById(eServiceId: String, updateEServiceSeed: UpdateEServiceSeed)(implicit
```

Comment:
please do not move code if not strictly necessary, it makes review and possible conflicts resolution more difficult 🙏